libvast/src/system/configuration.cpp

```diff
@@ -61,6 +61,14 @@ caf::error configuration::parse(int argc, char** argv) {
     for (const auto& [old, new_] : replacements)
       if (option == old)
         option = new_;
+  // Detect when running with --disable-configuration, and remove the option
+  // from the command line.
```

Comment:
Why do we need this block? Is it because CAF doesn't support "flag"-like command line options?

Reply:
The other options only get parsed after we have the command structure, which is too late to affect the plugin configuration. Hence the manual parsing here.